app/models/container_service.rb

```diff
@@ -1,5 +1,6 @@
 class ContainerService < ApplicationRecord
   include CustomAttributeMixin
+  include SupportsFeatureMixin
```

Comment:
I'm not sure why this is here?

As far as I know we should collect metrics on container entities if the associated ext has a metrics endpoint